domain/src/main/kotlin/no/nav/su/se/bakover/domain/oppdrag/Utbetalingsstrategi.kt

```diff
@@ -129,15 +133,50 @@ sealed class Utbetalingsstrategi {
             )
         }
 
-        private fun createUtbetalingsperioder(beregning: Beregning) =
-            SlåSammenEkvivalenteMånedsberegningerTilBeregningsperioder(beregning.getMånedsberegninger()).beregningsperioder
-                .map {
+        private fun createUtbetalingsperioder(
+            beregning: Beregning,
+            uføregrunnlag: List<Grunnlag.Uføregrunnlag>,
+        ): List<Utbetalingsperiode> {
+            val slåttSammenMånedsberegninger = SlåSammenEkvivalenteMånedsberegningerTilBeregningsperioder(
+                beregning.getMånedsberegninger(),
+            ).beregningsperioder
+
+            val slåttSammenUføregrunnlag =
+                uføregrunnlag.slåSammenPeriodeOgUføregrad()
+
+            val månedsperioder = slåttSammenMånedsberegninger.map {
+                it.periode
+            }.flatMap {
+                it.tilMånedsperioder()
+            }.distinct()
+
+            val uføregrunnlagPerioder = slåttSammenUføregrunnlag.map {
+                it.first
+            }.flatMap {
+                it.tilMånedsperioder()
+            }.distinct()
+
+            val uføregrunnlagInneholderAlleMånedsperioder = uføregrunnlagPerioder.containsAll(månedsperioder)
```

Comment:
```suggestion
            val uføregrunnlagInneholderAlleMånedsperioder = uføregrunnlagPerioder == månedsperioder
```
Burde ikke dette fungere?

Reply:
tror ikke dem kan være 1 til 1. Siden ved revurdering, vil vi ha en lenger uføregrunnlags-periode, enn månedsberegninger. eksempelvis at hvis vi har en stønadsperiode hele året. revurderer uførhet + inntekt siste 6 måender. da har vi en beregning med bare siste 6 måneder, og 1 år uføreperioder, som er delt i 2. 

Uføregrunnlagene da inneholder alle månedsberegningene, men ikke nødvendigvis omvendt